.code-samples.meilisearch.yaml

```diff
@@ -843,3 +843,6 @@ update_localized_attribute_settings_1: |-
   );
 reset_localized_attribute_settings_1: |-
   client.index("INDEX_NAME").resetLocalizedAttributesSettings();
+export_post_1: |-
```

Comment:
sorry for the delay.
can you reflec the example that is here: https://github.com/meilisearch/documentation/blob/f1562b2b98919c440f0fb124a5cfc8d66456e77b/.code-samples.meilisearch.yaml#L1470-L1481